R/databricks_connection.R

```diff
@@ -40,6 +40,11 @@ databricks_connection <- function(config, extensions) {
     }
   )
 
+  # Hand in driver's libPaths to worker if user hasn't already set libpaths for notebook-scoped libraries
```

Comment:
Can we add a unit test for this in https://github.com/sparklyr/sparklyr/blob/main/tests/testthat/test-databricks-connect.R?